examples/bri-3/src/bri/identity/bpiAccounts/agents/bpiAccountsStorage.agent.ts

```diff
@@ -38,8 +38,8 @@ export class BpiAccountStorageAgent extends PrismaService {
       bpiAccountModel.ownerBpiSubjectAccounts.map((o) => {
         return new BpiSubjectAccount(
```

Comment:
now that the prisma model and domain model are in sync, we can directly invoke mapper(o, BpiSubjectAccount,BpiSubjectAccount)

Reply:
i am pretty sure i got all of these in this PR https://github.com/eea-oasis/baseline/pull/585 but i will double check